### DISCUSSION TITLE
Saleor POS module / feature

### DISCUSSION THREAD

**Discussion by uc77** [mirumee/saleor-dashboard#5212] [General] [answered]
Dear Community,

I've been exploring open source commerce platforms and Saleor caught my attention since last year. That said it would be great to know if there are plans to have a POS module for instore payments that can work with the Storefront and backend modules.

Regards,
Jackie

**Comment by lkostrowski** [comment, marked as answer]
Hey

Saleor is a headless commerce, meaning it doesn't ship it's client-facing layers like storefront, or in this case - POS. 

You can create or integrate any type of POS you need via Saleor API. Technically such POS will act like an Saleor App - it will be able to access Saleor via API.
